components/ui/Navbar.js

```diff
@@ -32,24 +32,13 @@ export const Navbar = () => {
       <nav className={styles.navbar}>
         <div className={styles.navbarBG} />
         <ul className={styles.navbarList}>
-          {/* TASK - React 1 week 2 */}
+        
           {/* Create a <NavItem> component, which accepts the following:  */}
           {/* title, link, isActive  */}
-          <li className={classNames(styles.navbarLinks, {
-            [styles.isLinkActive]: navbarItems[0].link === currentPath,
-          })}>
-            <Link href={navbarItems[0].link}><b>01</b> {navbarItems[0].title}</Link>
-          </li>
-          <li className={classNames(styles.navbarLinks, {
-            [styles.isLinkActive]: navbarItems[1].link === currentPath,
-          })}>
-            <Link href={navbarItems[1].link}><b>02</b> {navbarItems[1].title}</Link>
-          </li>
-          <li className={classNames(styles.navbarLinks, {
-            [styles.isLinkActive]: navbarItems[2].link === currentPath,
-          })}>
-            <Link href={navbarItems[2].link}><b>03</b> NASA COLLABORATION</Link>
-          </li>
+          <NavItem title={navbarItems[0].title} link={navbarItems[0].link} isActive={navbarItems[0].link === currentPath} />
+          <NavItem title={navbarItems[1].title} link={navbarItems[1].link} isActive={navbarItems[1].link === currentPath} />
+          <NavItem title={navbarItems[2].title} link={navbarItems[2].link} isActive={navbarItems[2].link === currentPath} />
+      
```

Comment:
Here instead of calling NavItem multiple times you could rely on map to make it cleaner and more modular :)